black-mulberry-data-transfer/src/main/java/com/black/mulberry/data/transfer/model/CreateUserDto.java

```diff
@@ -0,0 +1,18 @@
+package com.black.mulberry.data.transfer.model;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+@Builder
+public class CreateUserDto {
+
+    private String name;
+    private String surname;
+    private String email;
+    private String password;
+
+}
```

Comment:
Why did you create these already exists then you don't need to re-create.

Reply:
what should be used instead of CrrentUserDto?